### DISCUSSION TITLE
Setup don't work

### DISCUSSION THREAD

**Discussion by dlsartori** [Pryaxis/TShock#2119] [General]
Why does setup don't work?
whithout it i can't get miself owner rank
and all tshock servers are the same thing

**Comment by Quinci135** [comment]
If you haven't already seen these, [here is the starting guide here on this repository](https://github.com/Pryaxis/TShock#new-to-tshock) and [here is the starting guide on the documentation site](https://tshock.readme.io/docs/getting-started)
Are you trying to do /setup on your own server? Does your server already have any accounts? If there are accounts, then setup will be disabled and you can change your group to owner by doing `/user group "your account name" owner` in the server console. Are you getting any errors in the console?
If you meant you were trying to do this command on other people's servers, no, that pretty much won't nor shouldn't work.
